packages/orbit-components/src/primitives/BadgePrimitive/index.jsx

```diff
@@ -73,8 +94,11 @@ const BadgePrimitive = (props: Props): React.Node => {
       ariaLabel={ariaLabel}
       borderColor={borderColor}
     >
+      {carriers && <CarrierLogo carriers={carriers} rounded size="medium" />}
```

Comment:
The spacing of the CarrierLogo doesn't look quite right. I used a red background so that you can see the problem.

<img width="268" alt="Screen Shot 2021-11-15 at 18 19 12" src="https://user-images.githubusercontent.com/471278/141825838-0ef1df00-c07b-45b3-8178-89f073317a27.png">